.github/workflows/update-authorized-users.yaml

```
@@ -1,22 +1,23 @@
 name: Update authorized users
 
 on:
-  workflow_dispatch:
-  schedule:
-    - cron: '30 21 * * 3'
+  pull_request_target:
+    types:
+      - closed
+    branches: 
+      - main
+    paths:
+      - 'model-metadata/*.yaml'
+
     
 jobs:
   update-users-list:
-    if: ${{ github.repository_owner == 'CDCgov' }}
+    if: ${{ github.repository_owner == 'CDCgov' &&  github.event.pull_request.merged == true }}
```

Don't need this if trigger is push, I think?
```suggestion
    if: ${{ github.repository_owner == 'CDCgov' }}
```